src/config/dark-sites.config

```diff
@@ -1,3 +1,4 @@
+*.itrade.gg
```

Comment:
Why you add it with `*.` in the beginning?
Is this site has some version with prefixes? If not then it should be added without that prefix!